doc/faq/howto_faq.rst

```diff
@@ -454,8 +453,7 @@ and :ref:`what-is-a-backend`).
 
 .. note::
    Because closing a figure window unregisters it from pyplot, you must call
-   `~matplotlib.pyplot.savefig` *before* calling ``show`` if you wish to save
-   the figure as well as view it.
+   `~matplotlib.pyplot.savefig` *before* calling ``show`` if you wish to save the figure as well as view it.
```

Comment:
Do we have line-length limits on .rst files?

Reply:
It's not strictly enforced because you sometimes need longer lines, e.g. for links or larger tables. However, you should wrap at 79 chars if possible.